.pre-commit-config.yaml

```diff
@@ -44,7 +44,6 @@ repos:
   hooks:
     - id: check-api-reference
       name: check-api-reference
-      pass_filenames: false
```

Comment:
Why remove this?

Reply:
Some mistake while going back and forth in the yaml - fixing now